src/script/calling/CallingRepository.ts

```diff
@@ -546,18 +546,32 @@ export class CallingRepository {
    * Handle incoming calling events from backend.
    */
   onCallEvent = async (event: CallingEvent, source: string): Promise<void> => {
-    const {content, conversation, qualified_conversation, from, qualified_from, sender: clientId, time} = event;
+    const {
+      content,
+      conversation,
+      qualified_conversation,
+      from,
+      qualified_from,
+      sender: clientId,
+      time = new Date().toISOString(),
+    } = event;
     const isFederated = this.core.backendFeatures.isFederated && qualified_conversation && qualified_from;
     const userId = isFederated ? qualified_from : {domain: '', id: from};
     const conversationId = isFederated ? qualified_conversation : {domain: '', id: conversation};
     const currentTimestamp = this.serverTimeHandler.toServerTimestamp();
     const toSecond = (timestamp: number) => Math.floor(timestamp / 1000);
     const contentStr = JSON.stringify(content);
+    const conversationEntity = this.conversationState.findConversation(conversationId);
```

Comment:
Let's get rid of the `<stuff>Entity` convention. Let's just call it `conversation`
```suggestion
    const conversation = this.conversationState.findConversation(conversationId);
```

Reply:
<img width="532" alt="image" src="https://user-images.githubusercontent.com/63250054/188905313-580939e9-6368-47cc-bc6f-76798d42aab5.png">